src/server/api/v1/v1Router.js

```diff
@@ -6,8 +6,13 @@ const logger = require("../../../backend/logwrapper");
 
 router.use(function log(req, res, next) {
     // here we could do stuff for every request if we wanted
+    const clientIp = req.socket.remoteAddress;
+    // check for proper ipv4 address (possible for ipv4 address in ipv6 format)
+    const matches = clientIp.match(/(\d+\.\d+\.\d+\.\d+)/);
```

Comment:
If anything, I'd want to know if a request went over ipv6 while being an ipv4 address, I've had dual-stack cause issues several times, and this change would make those more difficult to diagnose

Reply:
thats a solid point, I can remove the regex matching so that it just serves the raw remoteAddress. Through that enough information could be gathered to determine where the requests are coming from.